test/test_response.py

```diff
@@ -332,6 +332,21 @@ def test_decode_zstd(self) -> None:
         r = HTTPResponse(fp, headers={"content-encoding": "zstd"})
         assert r.data == b"foo"
 
+    @onlyZstd()
+    def test_decode_multiframe_zstd(self) -> None:
+        data = (
+            # Zstandard frame
+            zstd.compress(b"foo")
+            # skippable frame (must be ignored)
+            + bytes.fromhex("502A4D1810000000726f676468616d207761732068657265")
```

Comment:
The magic number is in little-endian, and the last half-byte (noted `?` in the spec) can be anything, I chose `0` here but it is arbitrary. So 0x184D2A50, which encoded little-endian is `502A4D18`.

Then followed by the frame size, I chose 16, so `10000000`.

Then the user data part, I used 16 arbitrary bytes.

Maybe it would be clearer if I put some spaces in there, like one of the following:
```python
bytes.fromhex("502A4D18 10000000 726f676468616d207761732068657265")
bytes.fromhex("50 2A 4D 18    10 00 00 00    72 6f 67 64 68 61 6d 20 77 61 73 20 68 65 72 65")
```

Tell me if that makes more sense or if you want me to change it to something else.

Reply:
I like option 3! Can we mention additionally that magic number and frame size are little-endian? And maybe use only zeroes for user data? Otherwise it sounds like the randomness is significant.

Reply:
I agree, let's add comments to the parts of the frame and use zeroes to denote insignificant data.

Reply:
Done! I have changed the frame size to 7 so that it is clearer that it does not need to be a round number.